caasp-istio-proxyv2-image/caasp-istio-proxyv2-image.kiwi

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v5/istio-proxyv2:%%PKG_VERSION%%-rev<VERSION> caasp/v5/istio-proxyv2:%%PKG_VERSION%%-rev<VERSION>-build -->
+
+<image schemaversion="6.9" name="caasp-istio-proxyv2-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Istio proxyv2 running on SLE15 SP2 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.2">
+      <containerconfig
+        name="caasp/v5/istio-proxyv2"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/pilot-agent proxy --binaryPath /usr/bin/envoy"/>
```

Comment:
the `proxy` arg is already provided by the manifest. 
The manifest also provides a `--binaryPath`, therefore, I suggest looking for an alternative

Reply:
I updated the image in IBS to have:
<entrypoint execute="/usr/bin/pilot-agent"/>

Reply:
Updated to `/usr/bin/pilot-agent`